srcpkgs/acpica-utils/template

```diff
@@ -9,7 +9,7 @@ license="GPL-2.0-only, BSD-3-Clause"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="https://www.acpica.org/"
 distfiles="https://acpica.org/sites/acpica/files/acpica-unix-${version}.tar.gz"
-checksum=2643911d0e74c52e4122b914dbbbfa8e2559e4414342bc45f268d2fae32c1ef3
+checksum=dc408d11889bcbedcfe9cc5b7ed23f65e857ca8fd5125f8c7a9e075e0b282db1
 only_for_archs="i686 i686-musl x86_64 x86_64-musl"
 
 do_build() {
```

Comment:
Maybe we could improve this template, by removing `do_build()` and `do_install()` completely:
````
# Template file for 'acpica-utils'
pkgname=acpica-utils
version=20180927
revision=1
wrksrc="acpica-unix-${version}"
build_style=gnu-makefile
make_use_env=yes
hostmakedepends="flex"
short_desc="Intel ACPI CA Unix utilities"
license="GPL-2.0-only, BSD-3-Clause"
maintainer="Juan RP <xtraeme@voidlinux.eu>"
homepage="https://www.acpica.org/"
distfiles="https://acpica.org/sites/acpica/files/acpica-unix-${version}.tar.gz"
checksum=dc408d11889bcbedcfe9cc5b7ed23f65e857ca8fd5125f8c7a9e075e0b282db1
only_for_archs="i686 i686-musl x86_64 x86_64-musl"

case "$XBPS_TARGET_MACHINE" in
	i686*) make_build_args="BITS=32";;
	*) make_build_args="BITS=64";;
esac

pre_build() {
	sed -e 's/_CYGWIN/_LINUX/g' -e 's/-Werror//g' -i generate/unix/Makefile.config generate/unix/iasl/Makefile
}
````